igor-core/src/main/java/com/netflix/spinnaker/igor/polling/CommonPollingMonitor.java

```diff
@@ -67,12 +71,9 @@ public CommonPollingMonitor(
     this.discoveryStatusListener = discoveryStatusListener;
     this.lockService = lockService;
     this.scheduler = scheduler;
+    this.instrumentation = new CommonPollingMonitorInstrumentation(registry);
```

Comment:
single instance per monitor, I know it's not needed, but given we just run a few, it's not a biggy, compared to fix all the tests and change the constructor of every subclass.
The key bit is the registry, which can be passed in for testing